### ISSUE TITLE
Status connected but buttons are not changing

### ISSUE THREAD

**Issue by eric-emc** [bitfocus/companion-satellite#24]
I'm using multiple companion remotes running on different raspberry pi models 2b and 3b. Unfortunately, we sometimes have network dropouts and the stream decks disconnect and display the connecting image on the buttons.
Usually it reconnects within a few seconds, but sometimes it connects and doesn't seem to reconnect the stream deck to the companion instance running on an intel nuc. 
Most of my companion remotes are linked by cable and one of them over wifi.
On the wifi model this sometimes happens when roaming between different access points. Usually after connecting to a 3rd access point it reconnects again.
Is there something I'm missing? Or do you have any idea why it sometimes connects as expected and sometimes it doesn't?
Thank you very much!

Companion: 2.1.2
Companion remote: 0.2.2
Node: 14.16.0

![disconnected](https://user-images.githubusercontent.com/82173039/114069594-7419c500-989f-11eb-861b-5e4f5fd4f77d.jpg)
![connected](https://user-images.githubusercontent.com/82173039/114069649-83990e00-989f-11eb-8054-d2c9a3648f73.jpg)

**Comment by Julusian** [comment]
I have seen this occasionally myself, but not often enough to find a pattern to it.
Generally it can be resolved by re-plugging the streamdeck

**Comment by Toshiba01** [comment]
I've had this issue as well, but I decided to update the "client.js" file with the folowing lines 

`const PING_UNACKED_LIMIT = 50; // Arbitrary number from 5(originally); 
const PING_INTERVAL = 200; //from 100(originally)`


That seem to solve my issues, where it was dropping the connection due to packet dropping. :)

**Comment by eric-emc** [comment]
I did some testing in the last few days and unfortunately this didn't fix it for me.
As I see it this just increases the time until the buttons change to the "connecting screen". I also noticed that during this time when the buttons are not connected to Companion they are freezed and the key presses don't work. This is actually even worse than the "connecting screen" because as a enduser I can't know if the buttons are connected or not.
It would be great if the buttons can switch back automatically. Re-plugging the streamdeck seems to work but this is a hassle because the USB ports are not in reachable spaces...
